### ISSUE TITLE
Demo Xcode 8.2.1 / Swift 3 (Unresolved identifier)

### ISSUE THREAD

**Issue by iLandes** [LISNR/LNRSimpleNotifications#23]
Hi there,

I'm trying to test your lib. With the demo project (after pod install) I have this issue from compiler :

`/LNRSimpleNotifications/LNRSimpleNotificationsDemo/LNRSimpleNotificationsDemo/ViewController.swift:56:60: Use of unresolved identifier 'LNRNotification'`

**Comment by Jon-Schneider** [comment]
Hmm. What version of the pod does your Podfile.lock have? It should be 0.6.0, but you may somehow have an older version. I would run 'pod update' and see if that fixes the issue.

**Comment by paranat** [comment]
Getting a similar error using Carthage with version 0.6.0.

```
*** Checking out LNRSimpleNotifications at "0.6.0"
[...]
*** Building scheme "LNRSimpleNotifications" in LNRSimpleNotifications.xcworkspace
[...]
[...]/LNRSimpleNotifications/Source/LNRNotificationView.swift:26:30: error: use of undeclared type 'LNRNotification'
[...]/LNRSimpleNotifications/Source/LNRNotificationView.swift:33:24: error: use of undeclared type 'LNRNotification'
[...]/LNRSimpleNotifications/Source/LNRNotificationView.swift:102:29: error: use of unresolved identifier 'LNRNotification'
[...]/LNRSimpleNotifications/Source/LNRNotificationManager.swift:38:48: error: use of undeclared type 'LNRNotification'
A shell task (/usr/bin/xcrun xcodebuild -workspace [...]/LNRSimpleNotifications/LNRSimpleNotifications/LNRSimpleNotifications.xcworkspace -scheme LNRSimpleNotifications -configuration Release -sdk iphoneos ONLY_ACTIVE_ARCH=NO BITCODE_GENERATION_MODE=bitcode CODE_SIGNING_REQUIRED=NO CODE_SIGN_IDENTITY= CARTHAGE=YES clean build) failed with exit code 65:```

**Comment by Jon-Schneider** [comment]
@iLandes @paranat This should be fixed now in 0.6.1. Can you please test it and let me know?

**Comment by paranat** [comment]
@Jon-Schneider Yep, it's building fine now as of 0.6.1. Thanks!
